contracts/cw1-subkeys/src/contract.rs

```diff
@@ -1179,10 +1179,10 @@ mod tests {
 
         // And then cannot (not enough funds anymore)
         let res = execute(deps.as_mut(), mock_env(), info, execute_msg.clone());
-        match res.unwrap_err() {
-            ContractError::Std(StdError::Underflow { .. }) => {}
-            e => panic!("unexpected error: {:?}", e),
-        }
+        assert!(matches!(
```

Comment:
This is definitely a step forward.

I wonder if this can somehow be compressed further. Could you experiment please? Maybe `let err = ....unwrap_err()` for the line above (chained with `execute`). Or maybe there is a shorthand for        `ContractError::Std(StdError::Underflow { .. })` (which I doubt).

It is fine as is, just wondering if there is a way to simplify further before we leave this as a pattern for all to copy.

Reply:
We can put the `unwrap_err()` in the line above, sure. I'll do it.